netbox/dcim/forms/bulk_import.py

```diff
@@ -1161,27 +1161,35 @@ def __init__(self, *args, **kwargs):
         else:
             self.fields['parent'].queryset = InventoryItem.objects.none()
 
-    def clean_component_name(self):
-        content_type = self.cleaned_data.get('component_type')
-        component_name = self.cleaned_data.get('component_name')
+    def clean(self):
```

Comment:
Need to call `super()` when overriding `clean()` on a form.
```suggestion
    def clean(self):
        super().clean()
```